### PR TITLE
:construction_worker: Add a github action to build this project

### PR DESCRIPTION
**Pull request by Michiels96**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
+    - run: yarn add vite
+    - run: yarn build
+    - uses: actions/upload-artifact@v3
+      with:
+        name: build-artifact
+        path: build/
```
Comment (M4gie) [inline .github/workflows/build.yml]: I forgot to mention, the result of the build command is folder called `dist` :wink:

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
+    - run: yarn add vite
```
Comment (M4gie) [inline .github/workflows/build.yml:19]: You have to install all deps to build the projet, just run `yarn install`

**Review by M4gie — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  #   types: [created, published]
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
```
Comment (Tifloz) [inline .github/workflows/build.yml]: Please put a LTS version of node

**Review by Tifloz — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - run: yarn install
+    - run: yarn build
+    - uses: actions/upload-artifact@v3
+      with:
+        name: dist-artifact
```
Comment (Tifloz) [inline .github/workflows/build.yml:23]: Cand you specify a better Candy name ? 😸

**Review by Tifloz — CHANGES_REQUESTED** [review]

**Review by M4gie — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (Michiels96) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml (hunk 3 above)
Comment (Michiels96) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml (hunk 4 above)
Comment (Michiels96) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml (hunk 2 above)
Comment (Michiels96) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    branches:
+    - main
+  release:
+    types: [created, published]
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```
Comment (Tifloz) [inline .github/workflows/build.yml:14]: Can you specify a name for each major step of your workflow ? To ensure to have correctly formatted logs

**Review by Tifloz — CHANGES_REQUESTED** [review]

**Pull request merged**